content/docs/details/decorating-components/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Introduction
+publishedDate: '2023-09-05'
+description: An introduction about decorating components.
+---
+
+Catalog is built around the concept of [entity descriptor YAML files](https://backstage.io/docs/features/software-catalog/descriptor-format/), therefore, any change you wish to make to your components, needs to be done via editing those files. 
+
+However, as YAML files can grow in complexity, they can become challenging to read and maintain. Editing those files, can sometimes be a long process, especially if ownership is shared accross the teams in your organization.
```

Comment:
It's likely better to take an approach where we talk about multiple sources of data.